WooCommerce/Classes/ViewRelated/Orders/Order Details/Product List Section/Fulfillment/Cells/PickListTableViewCell.swift

```diff
@@ -72,13 +72,45 @@ final class PickListTableViewCell: UITableViewCell {
         super.awakeFromNib()
 
         selectionStyle = .none
-
+        configureBackground()
         setupImageView()
         setupNameLabel()
         setupQuantityLabel()
         setupSkuLabel()
     }
+}
+
+
+/// MARK: - Public Methods
+///
+extension PickListTableViewCell {
+    func configure(item: OrderItemViewModel) {
+        if item.productHasImage,
+        let imageURL = item.imageURL {
+                productImageView.downloadImage(from: imageURL,
+                                               placeholderImage: UIImage.productPlaceholderImage)
+        } else {
+            productImageView.image = .productPlaceholderImage
+        }
+
+        name = item.name
+        quantity = item.quantity
+        sku = item.sku
```

Comment:
When I wrote this code I found it very awkward. I _still_ can't find a better way to configure an imageView. :|

Reply:
Is there any reason to unwrap `item.imageURL`?
It seems that `downloadImage()` allows optional URL, so if `imageURL` is nil, it automatically shows the placeholder.

The whole condition can thus be rewritten

```
productImageView.downloadImage(from: item.imageURL,
                                               placeholderImage: UIImage.productPlaceholderImage)
```

---
Maybe it can be further improved when we will implement the Image Caching https://github.com/woocommerce/woocommerce-ios/issues/1432

Reply:
+1  this looks better already.